lib/howl/log.moon

```diff
@@ -45,5 +51,6 @@ error = (message) -> last_error = dispatch 'error', message
 clear = ->
   entries = {}
   last_error = nil
+debug = (msg, ...) -> io.stderr\write '\n'..msg, serpent.block {...}
```

Comment:
The output of this isn't very nice, e.g. `log.debug "foo"` generates this:

foo{} --[[table: 0x41dd76d0]]

If we want to to keep it it should be improved.